servicecontrol/servicecontrol-instances/hardware.md

```diff
@@ -0,0 +1,61 @@
+---
+title: ServiceControl Hardware Considerations
+summary: Describes performance benchmarks performed for ServiceControl
+reviewed: 2018-10-15
+---
+
+ServiceControl as an application can be used to process the entire message load of a system. This articles provides general guidelines, recommendations, and performance benchmarks to help determine the resources to provide for a production environment. To identify the hardware specifications for any system environment a combination of testing with the system and the information provided below will need to be used.
+
+## General Recommendations
+
+* Use dedicated hardware for ServiceControl in production.
```

Comment:
I am a bit confused by today's terminology. Does the term `hardware` include VMs? I mean when I was young we used the term hardware to refer to bare metal. So this sentence can be understood as "don't use VMs for SC, use bare meta machines only"

Reply:
Good point, I think we want the recommendation to be for the user to assign a dedicated server. We will reword it.